### ISSUE TITLE
Sorting not working

### ISSUE THREAD

**Issue by flashoflight1** [toopriddy/mytime#385]
The sort by buttons in my time no longer work. 

Thanks 
Steve Mathey

**Comment by PAMELA3BLACK** [comment]
Sorting tabs don’t work on iphone 11 ProMax

**Comment by toopriddy** [comment]
Sorry for the trouble. This is a duplicate of https://github.com/toopriddy/mytime/issues/377 please see the workaround while we wait on Apple to fix the bug
